protected/humhub/modules/user/behaviors/Followable.php

```diff
@@ -208,16 +243,14 @@ public function getFollowingCount($objectModel)
      *
      * E.g. Get list of spaces which are the user follows.
      *
-     * @param CDbCriteria $eCriteria e.g. for limit the result
+     * @deprecated Use $this->getFollowingQuery()->all() instead
+     * @param ActiveQueryUser $query e.g. for limit the result
      * @param string $objectModel HActiveRecord Classname to restrict Object Classes to (e.g. User)
-     * @return Array
+     * @return User[]
```

Comment:
Also deprecate this one, I think its only used in `userFollower` view.

Reply:
I already wrote above this `* @deprecated Use $this->getFollowingQuery()->all() instead`, is it not enough for deprecation? If no, please let me know what I should do, thanks.

Reply:
Sorry, probably missed the phpdoc. But we should always add the version like `@deprecated since 1.9 some replacement infos`